### ISSUE TITLE
[med] Other A11y: Autocomplete attribute is not provided on the page.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8615]
## Action Performed:
1. Using JAWS + Chrome browser, open the URL staging.new.expensify.com/r/85021985                                                                                          2. Open settings and navigate to the workspace using Tab key.
3. Open workspace using Enter and go to help button.   
4. New content will appear navigate to the request a setup call using Tab key and press Enter.
5. Check whether the auto complete option is provided or not.

## Expected Result:
Any form elements that accept an individual's personal data should have the autocomplete attribute with the appropriate value to identify the purpose of that UI component

## Actual Result:
First name, Last name and Phone number field forms doesn't not have the autocomplete attribute.

HTML
`<input autocapitalize="sentences" autocomplete="on" autocorrect="on" dir="auto" spellcheck="true" type="text" class="css-11aywtz r-13awgt0 r-tz77pi r-13qz1uu" value="" name="fname" style="border-radius: 8px; border-width: 0px; box-shadow: none; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; outline-width: 0px; padding: 23px 11px 8px;">`

**Other Occurrences:**
Same issue repro on PR #8730, #8826, #8868, #8802

## Workaround:
Yes

## Area issue was found in:
Workspace -> Get assistance

## Failed WCAG checkpoints
1.3.5

## User impact:

https://user-images.githubusercontent.com/97600950/163106833-b43d664c-0029-4fad-8af9-56410b763551.mp4

People with cognitive disabilities can be disoriented when there are missing autocomplete attributes in the input fields that collect an individual's personal data.

## Suggested resolution:
Ensure providing the appropriate autocomplete attributes value. Consider providing 'First name, last name and phone number' respectively. 

Refer to:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-autocomplete

## Platform:

- Web
- iOS
- Android
- Desktop App

**Version Number:** v1.1.49-1
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Issue reported by:** Stuti
